entrypoint.sh

```diff
@@ -28,17 +28,12 @@ if [ -z "$WIKI_DIR" ]; then
     WIKI_DIR='wiki/'
 fi
 
-echo "Configuring wiki git..."
-mkdir $TEMP_CLONE_FOLDER
-cd $TEMP_CLONE_FOLDER
-git init
-
-# Setup credentials
-git config user.name $GH_NAME
-git config user.email $GH_MAIL
+# Disable Safe Repository checks
+git config --global --add safe.directory "/github/workspace"
+git config --global --add safe.directory "/github/workspace/$TEMP_CLONE_FOLDER"
 
-git pull https://$GH_TOKEN@github.com/$REPO.wiki.git
-cd ..
+echo "Cloning wiki git..."
+git clone https://$GH_TOKEN@github.com/$REPO.wiki.git $TEMP_CLONE_FOLDER
```

Comment:
I believe this should include both the GH_TOKEN and GH_NAME:GH_TOKEN as seen in the bottom of the file. We've had issues in the past when it was just the GH_TOKEN.

Reply:
I'm happy to add that, but I kept the URL exactly as you already had it for the `git pull` command. The same URL is also used for the `push` performed later on. Your previous code attempted to use both for the push, but I feel that only one should be used to keep things simple. I'll update it to include both as you ask.

Reply:
Ok change is made. Here is the Github Action I use in another personal project of mine to test my changes:
https://github.com/rcdailey/trash-updater/runs/6162111655?check_suite_focus=true